addons/crm/models/crm_recurring_plan.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class RecurringPlan(models.Model):
+    _name = "crm.recurring.plan"
+    _description = "Plan used to calculate recurring revenues"
+    _order = "sequence"
+
+    name = fields.Char('Plan Name', required=True, translate=True)
+    number_of_months = fields.Integer('# Months', required=True)
```

Comment:
Maybe can you add a SQL constrains to check that this value is positive and not null ?

```python
_sql_constraints = [
    ('check_number_of_months', 'CHECK(number_of_months > 0)', 'The number of month must be strictly positive'),
]
```